shotover-proxy/tests/test-configs/redis/cluster-auth/docker-compose.yaml

```diff
@@ -1,54 +1,54 @@
 services:
-  redis-node-0:
-    image: &image bitnami/redis-cluster:6.2.12-debian-11-r26
+  valkey-node-0:
+    image: &image bitnami/valkey-cluster:7.2.5
```

Comment:
can we specify the whole image name, e.g. `bitnami/valkey-cluster:7.2.5-debian-12-r3`
It helps prevent CI from breaking in case an issue or incompatibility occurs in a later release of the image.

Reply:
I believe this is a specific image and versioned pinned. I would think this will alway pull 7.2.5 and not 7.2.5-debian-12-r4 or any other version. Am I understanding this wrong?

Reply:
![Uploading Screenshot 2024-11-13 at 1.13.57 PM.png…]()

Reply:
I wanted to use the exact version as used in our managed platform...

Reply:
<img width="1462" alt="Screenshot 2024-11-13 at 1 13 57 PM" src="https://github.com/user-attachments/assets/14ab4c56-41b2-40aa-8717-51470bfae22d">

Reply:
![image](https://github.com/user-attachments/assets/fa34fdaf-f3dc-4408-a167-d2591b183d50)
Seeing as the hash of 7.2.5 matches the hash of 7.2.5-debian-12-r4 but not 7.2.5-debian-12-r3, I would have to assume that 7.2.5 tag is changed every time a new `-rN` release is made.

Reply:
> `I wanted to use the exact version as used in our managed platform`
Using any 7.2.5 release is going to be equivalent as to whats used in the managed platform.
The `-rN` at the end of the version is used for any changes to the bitnami image, say a more recent version of ubuntu packages or altering how the env vars used for config are processed. It does not indicate any change to the version of valkey that is being packaged.

Reply:
Ah. Got it. Will update.